### PR TITLE
New onboarding visual fixes

### PR DESCRIPTION
**Pull request by arjunu**
- fixed space between WalletChooser buttons
- fixed confirm recovery phrase dropdown borders not visible
- fixed account recovery dropdowns missing borders on page zoom change
- fixed spacing between logo & text in meta mask login error
- fixed spacing around ledger wallet error
- fixed metamask sign up wrong copy

### CODE REVIEW

File: packages/web/app/components/shared/forms/fields/VirtualizedSelect.modules.scss [hunk 1]
```diff
@@ -1,91 +0,0 @@
-//this is to sync the look of virtualizedSelect with NDS
```
Comment (arjunu) [inline packages/web/app/components/shared/forms/fields/VirtualizedSelect.modules.scss:1]: duplicate left over file

File: packages/web/app/components/wallet-selector/WalletRecovery/RecoverWallet/RecoverWalletCheckSeed/SeedRecovery.unsafe.module.scss [hunk 2]
```diff
@@ -34,12 +36,24 @@
...
+  top: 1px;
+  left: 30px;
+}
+
+:global(.Select-control) {
+  height: 28px;
+}
+
+:global(.Select-menu-outer) {
+  margin-top: 2px;
```
Comment (Moejoe90) [inline packages/web/app/components/wallet-selector/WalletRecovery/RecoverWallet/RecoverWalletCheckSeed/SeedRecovery.unsafe.module.scss:56]: can you use px-to-rem?

File: packages/web/app/components/wallet-selector/WalletSelectorLogin/LoginBrowserWallet/LoginBrowserWallet.tsx [hunk 3]
```diff
@@ -49,9 +52,13 @@ export const LoginBrowserWallet = compose(
     )(TransitionalLayout),
   ),
   withContainer(
-    withProps({
-      isLogin: true,
-    })(RegisterBrowserWalletContainer),
+    withProps<TWalletBrowserBaseExternalProps, { uiState: ECommonWalletRegistrationFlowState }>(
+      ({ uiState }) => ({
+        isLogin: true,
+        showWalletSelector: true,
```
Comment (Moejoe90) [inline packages/web/app/components/wallet-selector/WalletSelectorLogin/LoginBrowserWallet/LoginBrowserWallet.tsx:58]: the issue here is now you will always show the wallet selector and this is not correct in some cases. 
For example when the user signs etc `showWalletSelector` comes as a value from the state

File: packages/web/app/components/wallet-selector/WalletSelectorRegister/RegisterBrowserWallet/RegisterBrowserWallet.ts [hunk 4]
```diff
@@ -62,9 +65,13 @@ export const RegisterBrowserWallet = compose<IStateProps & IDispatchProps & TAdd
     renderComponent(LoadingIndicator),
   ),
   withContainer(
-    withProps({
-      isLogin: false,
-    })(RegisterBrowserWalletContainer),
+    withProps<TWalletBrowserBaseExternalProps, { uiState: ECommonWalletRegistrationFlowState }>(
+      ({ uiState }) => ({
+        isLogin: false,
+        showWalletSelector: true,
```
Comment (Moejoe90) [inline packages/web/app/components/wallet-selector/WalletSelectorRegister/RegisterBrowserWallet/RegisterBrowserWallet.ts:71]: same point mentioned in LoginBrowserWallet

File: packages/web/app/components/wallet-selector/WalletSelectorRegister/RegisterLedger/RegisterLedger.tsx [hunk 5]
```diff
@@ -49,11 +51,13 @@ export const RegisterLedger = compose<TWalletRegisterData, {}>(
...
   ),
   withContainer(
-    withProps<TWalletBrowserBaseProps, TWalletBrowserBaseProps>(({ showWalletSelector }) => ({
-      //MOE Check if this is needed
-      showWalletSelector,
-      isLogin: false,
-    }))(LedgerOnboardingContainer),
+    withProps<TLedgerContainerBaseProps, { showWalletSelector: boolean | undefined }>(
+      ({ showWalletSelector }) => ({
+        //MOE Check if this is needed
```
Comment (Moejoe90) [inline packages/web/app/components/wallet-selector/WalletSelectorRegister/RegisterLedger/RegisterLedger.tsx:56]: oops can we remove this 
`//MOE Check if this is needed`

File: packages/web/app/components/wallet-selector/shared/ledger-wallet/DefaultLedgerError/DefaultLedgerError.tsx [hunk 6]
```diff
@@ -18,7 +18,7 @@ export const DefaultLedgerError: React.FunctionComponent<TLedgerErrorProps> = ({
   errorMessage,
   tryToEstablishConnectionWithLedger,
 }) => (
-  <section className="text-center my-5">
+  <section className="text-center">
```
Comment (Moejoe90) [inline packages/web/app/components/wallet-selector/shared/ledger-wallet/DefaultLedgerError/DefaultLedgerError.tsx:21]: are you sure we don't need this?

Comment (arjunu) [reply packages/web/app/components/wallet-selector/shared/ledger-wallet/DefaultLedgerError/DefaultLedgerError.tsx:21]: yes it adds extra space that's not in the design

File: packages/web/app/components/wallet-selector/WalletRecovery/RecoverWallet/RecoverWalletCheckSeed/SeedRecovery.unsafe.module.scss (hunk 2 above)
Comment (arjunu) [reply packages/web/app/components/wallet-selector/WalletRecovery/RecoverWallet/RecoverWalletCheckSeed/SeedRecovery.unsafe.module.scss]: changed

File: packages/web/app/components/wallet-selector/WalletSelectorLogin/LoginBrowserWallet/LoginBrowserWallet.tsx (hunk 3 above)
Comment (arjunu) [reply packages/web/app/components/wallet-selector/WalletSelectorLogin/LoginBrowserWallet/LoginBrowserWallet.tsx]: removed

File: packages/web/app/components/wallet-selector/WalletSelectorRegister/RegisterBrowserWallet/RegisterBrowserWallet.ts (hunk 4 above)
Comment (arjunu) [reply packages/web/app/components/wallet-selector/WalletSelectorRegister/RegisterBrowserWallet/RegisterBrowserWallet.ts]: fixed

File: packages/web/app/components/wallet-selector/WalletSelectorRegister/RegisterLedger/RegisterLedger.tsx (hunk 5 above)
Comment (arjunu) [reply packages/web/app/components/wallet-selector/WalletSelectorRegister/RegisterLedger/RegisterLedger.tsx]: removed

**Pull request merged**